config/software/gmp.rb

```diff
@@ -17,6 +17,9 @@
 name "gmp"
 default_version "6.0.0a"
 
+# version_list: url=https://ftp.gnu.org/gnu/gmp/ filter=*.tar.bz2
+
+version("6.2.1")  { source md5: "28971fc21cf028042d4897f02fd355ea" }
```

Comment:
Can we update these SHAs to `sha256`? Thanks!

Reply:
Updated the sums from md5 to sha256